tests/Unit/Domain/DomainCreators/Test_Shell.cpp

```diff
@@ -419,7 +430,72 @@ SPECTRE_TEST_CASE("Unit.Domain.DomainCreators.Shell.Factory.ShellWedges",
+                              const bool use_logarithmic_map,
+                              const size_t radial_block_layers,
+                              const std::vector<size_t>& expected_block_ids) {
+  const size_t number_of_divisions =
+      radial_block_layers * pow(2, refinement_level);
+  const std::array<size_t, 2> grid_points_r_angular{{2, 3}};
+  const bool use_equiangular_map = true;
+  const double aspect_ratio = 1.0;
+  const ShellWedges which_wedges = ShellWedges::OneAlongMinusX;
+  // Points that lie on the interior of Blocks/Elements:
+  DataVector x_interior(number_of_divisions + 1);
+  // Points that lie on the boundary of Blocks/Elements:
+  DataVector x_boundary(number_of_divisions + 1);
+  for (size_t i = 0; i < number_of_divisions + 1; i++) {
+    const double delta =
+        static_cast<double>(i) / static_cast<double>(number_of_divisions);
+    if (use_logarithmic_map) {
+      x_interior[i] = inner_radius * pow(outer_radius / inner_radius, delta);
+      x_boundary[i] = (inner_radius - 0.1) *
+                      pow((outer_radius + 0.1) / (inner_radius - 0.1), delta);
+    } else {
+      x_interior[i] = inner_radius + delta * (outer_radius - inner_radius);
+      x_boundary[i] = (inner_radius - 0.1) +
+                      delta * ((outer_radius + 0.1) - (inner_radius - 0.1));
+    }
+  }
+  const auto zero = make_with_value<DataVector>(x_interior, 0.0);
+  tnsr::I<DataVector, 3, Frame::Inertial> interior_inertial_coords{
+      {{-x_interior, zero, zero}}};
+  const DomainCreators::Shell<Frame::Inertial> shell{
+      inner_radius - 0.1,    outer_radius + 0.1,  refinement_level,
+      grid_points_r_angular, use_equiangular_map, aspect_ratio,
+      use_logarithmic_map,   which_wedges,        radial_block_layers};
+  auto domain = shell.create_domain();
+  const auto blogical_coords =
+      block_logical_coordinates(domain, interior_inertial_coords);
+  for (size_t s = 0; s < expected_block_ids.size(); ++s) {
+    CHECK(blogical_coords[s].id.get_index() == expected_block_ids[s]);
+  }
+  size_t element_count = 0;
+  for (const auto& block : domain.blocks()) {
+    const auto initial_ref_levs = shell.initial_refinement_levels()[block.id()];
+    const std::vector<ElementId<3>> element_ids =
+        initial_element_ids(block.id(), initial_ref_levs);
+    for (const auto& element_id : element_ids) {
+      const auto element = create_initial_element(element_id, block);
+      if (element_id.segment_ids()[0] == SegmentId{refinement_level, 0} and
+          element_id.segment_ids()[1] == SegmentId{refinement_level, 0}) {
+        element_count++;
+        const auto map = ElementMap<3, Frame::Inertial>{
+            element_id, block.coordinate_map().get_clone()};
+        const tnsr::I<double, 3, Frame::Logical> logical_point(
+            std::array<double, 3>{{0.0, 0.0, 1.0}});
+        CHECK(magnitude(map(logical_point)).get() ==
+              approx(x_boundary[element_count]));
+      }
+    }
+  }
+}
```

Comment:
I am too tired to figure out what you have done here, but it does not appear to be what I was thinking I asked for.  I wanted you to create two Domains at the same time (two with log maps, and two without, like you do below, but with the same inner and outer boundary for the 8-layered unrefined, and single-layered refined Domains).  Then compute the radius (i.e. magnitude of the global coords) of an appropriate point in the layered unrefined Domain, and compare it to the radius of an appropriate point in the single-layered refined Domain.

Reply:
Ah, I think I might have had something like that before pushing, but then the same-ness of the boundaries and test points got lost when I refactored the test function - I will fix this!

Reply:
okay, if that is what it is doing, it should be fine.  you might want to put in a few comments describing what the purpose of the test is